libpod/events.go

```diff
@@ -0,0 +1,53 @@
+package libpod
+
+import (
+	"github.com/containers/libpod/libpod/events"
+	"github.com/hpcloud/tail"
+	"github.com/sirupsen/logrus"
+)
+
+// newContainerEvent creates a new event based on a container
+func (c *Container) newContainerEvent(status events.Status) {
+	e := events.NewEvent(status)
+	e.ID = c.ID()
+	e.Name = c.Name()
+	e.Image = c.config.RootfsImageName
+	e.Type = events.Container
+	if err := e.Write(c.runtime.config.EventsLogFilePath); err != nil {
+		logrus.Infof("unable to write event log to %s", c.runtime.config.EventsLogFilePath)
+	}
+}
+
+// newPodEvent creates a new event for a libpod pod
+func (p *Pod) newPodEvent(status events.Status) {
+	e := events.NewEvent(status)
+	e.ID = p.ID()
+	e.Name = p.Name()
+	e.Type = events.Pod
+	if err := e.Write(p.runtime.config.EventsLogFilePath); err != nil {
+		logrus.Infof("unable to write event log to %s", p.runtime.config.EventsLogFilePath)
+	}
+}
+
+// newVolumeEvent creates a new event for a libpod volume
+func (v *Volume) newVolumeEvent(status events.Status) {
+	e := events.NewEvent(status)
+	e.ID = v.Name()
```

Comment:
Leave this blank - volumes don't have an ID and we can't pretend they do

Reply:
SGTM